keyboards/keychron/v6/iso_encoder/iso_encoder.c

```diff
@@ -176,3 +176,17 @@ led_config_t g_led_config = {
 };
 
 #endif // RGB_MATRIX_ENABLE
+
```

Comment:
Above comment

Reply:
Removed the callback from `keyboards/keychron/v6/v6.c`.